src/ServiceManagement/Compute/Commands.ServiceManagement/Migration/MoveAzureReservedIP.cs

```diff
@@ -0,0 +1,79 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.WindowsAzure.Management.Network;
+using System.Management.Automation;
+
+namespace Microsoft.WindowsAzure.Commands.ServiceManagement.IaaS.Network
+{
+    /// <summary>
+    /// Migrate ASM ReservedIP to ARM
+    /// </summary>
+    [Cmdlet(VerbsCommon.Move, "AzureReservedIP"), OutputType(typeof(OperationStatusResponse))]
+    public class MoveReservedIPCommand : MoveAzureNetworkResourceBase
+    {
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "ReservedIP name")]
+        [ValidateNotNullOrEmpty]
+        public string ReservedIPName
+        {
+            get;
+            set;
+        }
+
+        protected override void OnProcessRecord()
+        {
+            ServiceManagementProfile.Initialize();
+
+            if (this.Validate.IsPresent)
```

Comment:
why do we need to protect these executions? this is not like a delete operation and dont need a confirmation for customers.

also, this is very similar to the other Move-* cmdlets (so wouldnt want to introduce breaking change/break uniformity unless required)